src/app/register/register.component.html

```diff
@@ -12,9 +12,14 @@
           text="Donor Registration"
         ></biggive-heading>
       </div>
+      @if (redirectPath.startsWith('regular-giving')) {
+        <p>Please register for a a Big Give donation account to set up your regular giving agreement.</p>
+        <p>You will be able to log-in in future to see your regular or cancel the agreement.</p>
```

Comment:
```suggestion
        <p>You will be able to log in any time to see or cancel the agreement.</p>
```
Maybe tweak roughly like this to avoid "in in"? Also fixes a typo.

Reply:
Yep, done in 90d31af0